src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BasicValidationServiceTest {
+
+    BasicValidationService validationService;
+
+    @BeforeEach
+    void init() {
+        validationService = new BasicValidationService();
+    }
+    @Test
+    public void validateAmount() {
+        Exception exception = assertThrows(IllegalArgumentException.class, () -> validationService.validateAmount(null));
+        assertEquals(exception.getMessage(), "Amount must not be null");
+        exception = assertThrows(IllegalArgumentException.class, () -> validationService.validateAmount(-5.7));
+        assertEquals(exception.getMessage(), "Amount must be greater than 0");
```

Comment:
Same, you check here 2 different test cases, separate them

Reply:
done